src/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -103,6 +105,7 @@ public void IntPtrFieldSetValue(int value)
 
         [Theory]
         [MemberData(nameof(Pointers))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Pointers through Invoke not implemented: https://github.com/dotnet/corert/issues/2113")]
```

Comment:
I'm ok with tracking items in TFS as long as no one comes back and complains to me about putting TFS bug #'s in open source comments.  I vaguely recall that practice was discouraged in MS code in the open.

In any case, this particular issue affects CoreRT as well as Project N.